index.html

```diff
@@ -1,538 +1,571 @@
+        <option value="SGS">South Georgia And The South Sandwich Islands</option>
+        <option value="SSD">South Sudan</option>
+        <option value="ESP">Spain</option>
+        <option value="LKA">Sri Lanka</option>
+        <option value="SDN">Sudan</option>
+        <option value="SUR">Suriname</option>
+        <option value="SJM">Svalbard And Jan Mayen</option>
+        <option value="SWZ">Swaziland</option>
+        <option value="SWE">Sweden</option>
+        <option value="CHE">Switzerland</option>
+        <option value="SYR">Syrian Arab Republic</option>
+        <option value="TWN">Taiwan</option>
+        <option value="TJK">Tajikistan</option>
+        <option value="TZA">Tanzania, United Republic Of</option>
+        <option value="THA">Thailand</option>
+        <option value="TLS">Timor-Leste, Democratic Republic of</option>
+        <option value="TGO">Togo</option>
+        <option value="TON">Tonga</option>
+        <option value="TTO">Trinidad And Tobago</option>
+        <option value="TUN">Tunisia</option>
+        <option value="TUR">Turkey</option>
+        <option value="TKM">Turkmenistan</option>
+        <option value="TCA">Turks And Caicos Islands</option>
+        <option value="TUV">Tuvalu</option>
+        <option value="UGA">Uganda</option>
+        <option value="UKR">Ukraine</option>
+        <option value="ARE">United Arab Emirates</option>
+        <option value="GBR">United Kingdom</option>
+        <option value="USA" selected>United States</option>
+        <option value="URY">Uruguay</option>
+        <option value="UZB">Uzbekistan</option>
+        <option value="VUT">Vanuatu</option>
+        <option value="VAT">Vatican City State</option>
+        <option value="VEN">Venezuela, Bolivarian Republic Of</option>
+        <option value="VNM">Vietnam</option>
+        <option value="VGB">Virgin Islands (British)</option>
+        <option value="VIR">Virgin Islands (US)</option>
+        <option value="WLF">Wallis And Futuna</option>
+        <option value="ESH">Western Sahara</option>
+        <option value="YEM">Yemen</option>
+        <option value="ZMB">Zambia</option>
+        <option value="ZWE">Zimbabwe</option>
+      </select>
+
+      <label>Choose search type</label>
+      <div class="search-type-dropdown">
+        <label class="search-type-selector search-type-selected" for="autocomplete-radio"
+          data-panel-type="autocomplete">
+          <div class="panel-content">
+            <h5 class="header">Predict the address as I type</h5>
+            <input type="radio" name="search-type-radio" id="autocomplete-radio" value="autocomplete" checked>
+            <a href="https://docs.experianaperture.io/address-validation/experian-address-validation/address-search-types/autocomplete/"
+              target="_blank">Autocomplete</a> search is designed to provide accurate suggestions in real-time. You can
+            also use what3words as input.

```

Comment:
Why did you split this on multiple rows ?

Reply:
fixed